fix_: match escaped JSON string when remove sensitive string

qfrank opened the pull request:
There's a chance we get an escaped JSON string after [marshalData](https://github.com/status-im/status-go/blob/27ad41ba8f500012e9e74b58e925d2de02b0b883/mobile/callogstatus_request_log.go#L176), this PR improved the regular expression to be compatible with matching JSON strings that contain escape characters.

fix relate [issue](https://discord.com/channels/1210237582470807632/1217172931197800468/1352672523434721290)

qfrank committed:
fix_: match escaped JSON string when remove sensitive string

status-im-auto commented:
### Jenkins Builds
<details>
<summary>Click to see older builds (68)</summary>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | fc5ed065 | [#1](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/1/) | 2025-03-24 01:39:57 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250324-013709-fc5ed0-pr6450.zip) |
| :heavy_check_mark: | fc5ed065 | [#1](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/1/) | 2025-03-24 01:40:08 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250324-013708-fc5ed0-pr6450.aar) |
| :heavy_check_mark: | fc5ed065 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/1/) | 2025-03-24 01:41:02 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250324-013712-fc5ed0-pr6450.zip) |
| :heavy_check_mark: | fc5ed065 | [#1](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/1/) | 2025-03-24 01:42:08 | ~4 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250324-013717-fc5ed0-pr6450.zip) |
| :heavy_check_mark: | fc5ed065 | [#1](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/1/) | 2025-03-24 01:42:48 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250324-013714-fc5ed0-pr6450.zip) |
| :heavy_check_mark: | fc5ed065 | [#1](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6450/1/) | 2025-03-24 01:43:11 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250324-013710-fc5ed0-pr6450.zip) |
| :heavy_check_mark: | fc5ed065 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/1/) | 2025-03-24 02:09:23 | ~32 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/1//consoleText) |
| :heavy_check_mark: | fc5ed065 | [#1](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/1/) | 2025-03-24 02:12:46 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/1//consoleText) |
| | | | | | | |
| :heavy_check_mark: | b78d91cd | [#2](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/2/) | 2025-03-24 10:23:11 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250324-102034-b78d91-pr6450.zip) |
| :heavy_check_mark: | b78d91cd | [#2](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/2/) | 2025-03-24 10:23:38 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250324-102033-b78d91-pr6450.aar) |
| :heavy_check_mark: | b78d91cd | [#2](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/2/) | 2025-03-24 10:23:53 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250324-102042-b78d91-pr6450.zip) |
| :heavy_check_mark: | b78d91cd | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/2/) | 2025-03-24 10:24:17 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250324-102038-b78d91-pr6450.zip) |
| :heavy_check_mark: | b78d91cd | [#2](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6450/2/) | 2025-03-24 10:25:58 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250324-102036-b78d91-pr6450.zip) |
| :heavy_check_mark: | b78d91cd | [#2](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/2/) | 2025-03-24 10:26:29 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250324-102038-b78d91-pr6450.zip) |
| :heavy_check_mark: | b78d91cd | [#2](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/2/) | 2025-03-24 10:53:20 | ~32 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/2//consoleText) |
| :heavy_check_mark: | b78d91cd | [#2](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/2/) | 2025-03-24 10:56:02 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/2//consoleText) |
| | | | | | | |
| :heavy_check_mark: | c3155fe7 | [#3](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/3/) | 2025-03-25 07:11:21 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250325-070845-c3155f-pr6450.zip) |
| :heavy_check_mark: | c3155fe7 | [#3](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/3/) | 2025-03-25 07:11:35 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250325-070844-c3155f-pr6450.aar) |
| :heavy_check_mark: | c3155fe7 | [#3](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/3/) | 2025-03-25 07:12:07 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250325-070854-c3155f-pr6450.zip) |
| :heavy_check_mark: | c3155fe7 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/3/) | 2025-03-25 07:12:27 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-070848-c3155f-pr6450.zip) |
| :heavy_check_mark: | c3155fe7 | [#3](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/3/) | 2025-03-25 07:14:18 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-070850-c3155f-pr6450.zip) |
| :heavy_check_mark: | c3155fe7 | [#3](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6450/3/) | 2025-03-25 07:15:19 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250325-070847-c3155f-pr6450.zip) |
| :heavy_check_mark: | c3155fe7 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/3/) | 2025-03-25 07:39:32 | ~30 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/3//consoleText) |
| :heavy_check_mark: | c3155fe7 | [#3](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/3/) | 2025-03-25 07:43:45 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/3//consoleText) |
| | | | | | | |
| :heavy_check_mark: | f769a24b | [#4](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/4/) | 2025-03-25 07:17:00 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250325-071424-f769a2-pr6450.zip) |
| :heavy_check_mark: | f769a24b | [#4](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/4/) | 2025-03-25 07:17:14 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250325-071422-f769a2-pr6450.aar) |
| :heavy_check_mark: | f769a24b | [#4](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/4/) | 2025-03-25 07:17:41 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250325-071431-f769a2-pr6450.zip) |
| :heavy_check_mark: | f769a24b | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/4/) | 2025-03-25 07:17:58 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-071426-f769a2-pr6450.zip) |
| :heavy_check_mark: | f769a24b | [#4](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/4/) | 2025-03-25 07:19:49 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-071427-f769a2-pr6450.zip) |
| | | | | | | |
| :heavy_check_mark: | b974298f | [#5](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/5/) | 2025-03-25 07:19:44 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250325-071705-b97429-pr6450.zip) |
| :heavy_check_mark: | b974298f | [#5](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/5/) | 2025-03-25 07:20:15 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250325-071723-b97429-pr6450.aar) |
| :heavy_check_mark: | b974298f | [#5](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/5/) | 2025-03-25 07:20:49 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250325-071744-b97429-pr6450.zip) |
| :heavy_check_mark: | b974298f | [#4](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6450/4/) | 2025-03-25 07:20:54 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250325-071530-b97429-pr6450.zip) |
| :heavy_check_mark: | b974298f | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/5/) | 2025-03-25 07:21:35 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-071802-b97429-pr6450.zip) |
| :heavy_check_mark: | b974298f | [#5](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/5/) | 2025-03-25 07:25:21 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-071958-b97429-pr6450.zip) |
| :heavy_multiplication_x: | b974298f | [#4](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/4/) | 2025-03-25 08:00:48 | ~21 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/4//consoleText) |
| :heavy_check_mark: | b974298f | [#4](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/4/) | 2025-03-25 08:18:37 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/4//consoleText) |
| | | | | | | |
| :heavy_check_mark: | fdb32546 | [#6](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/6/) | 2025-03-25 08:12:53 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250325-081013-fdb325-pr6450.zip) |
| :heavy_check_mark: | fdb32546 | [#6](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/6/) | 2025-03-25 08:13:15 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250325-081011-fdb325-pr6450.aar) |
| :heavy_check_mark: | fdb32546 | [#6](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/6/) | 2025-03-25 08:13:24 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250325-081020-fdb325-pr6450.zip) |
| :heavy_check_mark: | fdb32546 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/6/) | 2025-03-25 08:13:53 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-081015-fdb325-pr6450.zip) |
| :heavy_check_mark: | fdb32546 | [#6](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/6/) | 2025-03-25 08:15:36 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-081017-fdb325-pr6450.zip) |
| :heavy_check_mark: | fdb32546 | [#5](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6450/5/) | 2025-03-25 08:15:45 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250325-081014-fdb325-pr6450.zip) |
| :heavy_check_mark: | fdb32546 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/5/) | 2025-03-25 08:39:45 | ~29 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/5//consoleText) |
| | | | | | | |
| :heavy_check_mark: | e4e9f507 | [#7](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/7/) | 2025-03-25 08:15:36 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250325-081258-e4e9f5-pr6450.zip) |
| :heavy_check_mark: | e4e9f507 | [#7](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/7/) | 2025-03-25 08:16:11 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250325-081320-e4e9f5-pr6450.aar) |
| :heavy_check_mark: | e4e9f507 | [#7](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/7/) | 2025-03-25 08:16:27 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250325-081327-e4e9f5-pr6450.zip) |
| :heavy_check_mark: | e4e9f507 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/7/) | 2025-03-25 08:17:35 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-081357-e4e9f5-pr6450.zip) |
| :heavy_check_mark: | e4e9f507 | [#7](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/7/) | 2025-03-25 08:20:55 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-081541-e4e9f5-pr6450.zip) |
| :heavy_check_mark: | e4e9f507 | [#6](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6450/6/) | 2025-03-25 08:21:11 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250325-081549-e4e9f5-pr6450.zip) |
| :heavy_check_mark: | e4e9f507 | [#5](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/5/) | 2025-03-25 08:54:08 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/5//consoleText) |
| :heavy_check_mark: | e4e9f507 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/6/) | 2025-03-25 09:09:13 | ~29 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 66b77f33 | [#8](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/8/) | 2025-03-25 09:12:08 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250325-090931-66b77f-pr6450.zip) |
| :heavy_check_mark: | 66b77f33 | [#8](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/8/) | 2025-03-25 09:12:22 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250325-090929-66b77f-pr6450.aar) |
| :heavy_check_mark: | 66b77f33 | [#8](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/8/) | 2025-03-25 09:12:58 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250325-090937-66b77f-pr6450.zip) |
| :heavy_check_mark: | 66b77f33 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/8/) | 2025-03-25 09:13:09 | ~3 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-090932-66b77f-pr6450.zip) |
| :heavy_check_mark: | 66b77f33 | [#8](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/8/) | 2025-03-25 09:14:54 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-090934-66b77f-pr6450.zip) |
| :heavy_check_mark: | 66b77f33 | [#7](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6450/7/) | 2025-03-25 09:15:01 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250325-090932-66b77f-pr6450.zip) |
| :heavy_check_mark: | 66b77f33 | [#7](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/7/) | 2025-03-25 09:38:41 | ~29 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/7//consoleText) |
| :heavy_multiplication_x: | 66b77f33 | [#6](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/6/) | 2025-03-25 09:43:41 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/6//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 359db1ea | [#9](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/9/) | 2025-03-25 09:38:46 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250325-093549-359db1-pr6450.aar) |
| :heavy_check_mark: | 359db1ea | [#9](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/9/) | 2025-03-25 09:39:02 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250325-093550-359db1-pr6450.zip) |
| :heavy_check_mark: | 359db1ea | [#9](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/9/) | 2025-03-25 09:39:06 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250325-093558-359db1-pr6450.zip) |
| :heavy_check_mark: | 359db1ea | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/9/) | 2025-03-25 09:40:32 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-093553-359db1-pr6450.zip) |
| :heavy_check_mark: | 359db1ea | [#9](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/9/) | 2025-03-25 09:41:16 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-093555-359db1-pr6450.zip) |
| :heavy_check_mark: | 359db1ea | [#8](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6450/8/) | 2025-03-25 09:41:55 | ~6 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250325-093551-359db1-pr6450.zip) |
| :heavy_multiplication_x: | 359db1ea | [#7](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/7/) | 2025-03-25 09:47:06 | ~3 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/7//consoleText) |
| :heavy_check_mark: | 359db1ea | [#8](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/8/) | 2025-03-25 10:08:08 | ~29 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/8//consoleText) |
</details>

| :grey_question: | Commit | :hash: | Finished (UTC) | Duration | Platform | Result |
|-|-|-|-|-|-|-|
| :heavy_check_mark: | f0f0417a | [#10](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/10/) | 2025-03-25 10:02:12 | ~2 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250325-095921-f0f041-pr6450.aar) |
| :heavy_check_mark: | f0f0417a | [#10](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/10/) | 2025-03-25 10:02:22 | ~2 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250325-095922-f0f041-pr6450.zip) |
| :heavy_check_mark: | f0f0417a | [#10](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/10/) | 2025-03-25 10:02:32 | ~2 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250325-095931-f0f041-pr6450.zip) |
| :heavy_check_mark: | f0f0417a | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/10/) | 2025-03-25 10:03:57 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-095925-f0f041-pr6450.zip) |
| :heavy_check_mark: | f0f0417a | [#9](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6450/9/) | 2025-03-25 10:04:44 | ~5 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250325-095924-f0f041-pr6450.zip) |
| :heavy_check_mark: | f0f0417a | [#10](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/10/) | 2025-03-25 10:04:46 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-095927-f0f041-pr6450.zip) |
| :heavy_check_mark: | f0f0417a | [#8](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/8/) | 2025-03-25 10:34:26 | ~34 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/8//consoleText) |
| :heavy_check_mark: | f0f0417a | [#9](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/9/) | 2025-03-25 10:42:28 | ~34 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/9//consoleText) |
| | | | | | | |
| :heavy_check_mark: | 8adab6d5 | [#11](https://ci.status.im/job/status-go/job/prs/job/android/job/PR-6450/11/) | 2025-03-25 13:25:34 | ~3 min | `android` | [:package:`aar`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-android-250325-132229-8adab6-pr6450.aar) |
| :heavy_check_mark: | 8adab6d5 | [#11](https://ci.status.im/job/status-go/job/prs/job/ios/job/PR-6450/11/) | 2025-03-25 13:25:35 | ~3 min | `ios` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-ios-250325-132230-8adab6-pr6450.zip) |
| :heavy_check_mark: | 8adab6d5 | [#11](https://ci.status.im/job/status-go/job/prs/job/windows/job/x86_64/job/main/job/PR-6450/11/) | 2025-03-25 13:25:49 | ~3 min | `windows` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-windows-250325-132238-8adab6-pr6450.zip) |
| :heavy_check_mark: | 8adab6d5 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/aarch64/job/main/job/PR-6450/11/) | 2025-03-25 13:27:22 | ~4 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-132234-8adab6-pr6450.zip) |
| :heavy_check_mark: | 8adab6d5 | [#11](https://ci.status.im/job/status-go/job/prs/job/macos/job/x86_64/job/main/job/PR-6450/11/) | 2025-03-25 13:27:52 | ~5 min | `macos` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-macos-250325-132234-8adab6-pr6450.zip) |
| :heavy_check_mark: | 8adab6d5 | [#10](https://ci.status.im/job/status-go/job/prs/job/linux/job/x86_64/job/main/job/PR-6450/10/) | 2025-03-25 13:30:27 | ~7 min | `linux` | [:package:`zip`](https://status-im-prs.ams3.cdn.digitaloceanspaces.com/status-go-linux-250325-132232-8adab6-pr6450.zip) |
| :heavy_check_mark: | 8adab6d5 | [#10](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/10/) | 2025-03-25 13:51:46 | ~29 min | `tests-rpc` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests-rpc/job/PR-6450/10//consoleText) |
| :heavy_check_mark: | 8adab6d5 | [#9](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/9/) | 2025-03-25 13:58:34 | ~35 min | `tests` | [:page_facing_up:`log`](https://ci.status.im/job/status-go/job/prs/job/tests/job/PR-6450/9//consoleText) |

qfrank committed:
feat(callog)_: add definition sensitiveMethod to avoid being logged

- Added `status_request_log.go` and `status_request_log_test.go` for logging function calls and removing sensitive information from logs.
- Updated `call_raw.go` to utilize the new logging functionality for RPC calls.
- Refactored `methodAndParamsFromBody` to return structured data for better handling.
- Adjusted imports in `status.go` and `signals.go` to use the new logging package.

igor-sirotin commented on logutils/callog/status_request_log.go:
```diff
@@ -141,10 +147,20 @@ func Recover(logger *zap.Logger) {
+	if _, ok := sensitiveMethod[method]; ok {
+		return true
+	}
+	return false
```
bruh 😄 
```suggestion
	_, ok := sensitiveMethod[method]; ok
    return ok
```

qfrank committed:
chore_: log sensitive method without request and response

igor-sirotin commented on logutils/callog/status_request_log.go:
```diff
@@ -43,10 +45,14 @@ var sensitiveKeys = []string{
 var sensitiveRegex = regexp.MustCompile(sensitiveRegexString)
 
+var sensitiveMethod = map[string]bool{
+	"accounts_importMnemonic": true,
```
What about other method in `accounts` service? https://github.com/status-im/status-go/blob/2ced83abb7ef30762fd8e2f8a35ca90ba1163df2/services/accounts/accounts.go#L336 https://github.com/status-im/status-go/blob/2ced83abb7ef30762fd8e2f8a35ca90ba1163df2/services/accounts/accounts.go#L437 https://github.com/status-im/status-go/blob/2ced83abb7ef30762fd8e2f8a35ca90ba1163df2/services/accounts/accounts.go#L473 https://github.com/status-im/status-go/blob/2ced83abb7ef30762fd8e2f8a35ca90ba1163df2/services/accounts/accounts.go#L490

There are many endpoints accepting `mnemonic`, `password`, `privateKey`, etc.

qfrank replied:
I've added more ...
